src/nodes/Navigation/modes/TurntableNavigation.js

```diff
@@ -68,40 +71,48 @@ x3dom.TurntableNavigation.prototype.onDrag = function(view, x, y, buttonState)
 
 x3dom.TurntableNavigation.prototype.pan = function(view, tx, ty)
 {
-    if(this.target == null)
-        return;
+    if(this.target != null)
+    {            
+        var target = this.target;        
+        var bbox  = target._x3domNode.getVolume();
         
-    var target = this.target;        
-    var bbox  = target._x3domNode.getVolume();
-    
-    var viewpoint = view._scene.getViewpoint();
-    
-    view._up   = view._flyMat.e1();
-    view._from = view._flyMat.e3(); // eye
-    var s = view._flyMat.e0();                   
-
-    // add xy offset to look-at position  ^
-    cor = view._at;     
-                
-    cor = cor.addScaled(new x3dom.fields.SFVec3f(0,1,0), ty);
-    var temp = cor;
-    if(cor.y > bbox.max.y || cor.y < bbox.min.y )
-        temp = view._at;
-    else
-        view._from = view._from.addScaled(new x3dom.fields.SFVec3f(0,1,0), ty);
-    
-    
-    cor = temp.addScaled(new x3dom.fields.SFVec3f(1,0,0), tx); 
-    if(cor.x > bbox.max.x || cor.x < bbox.min.x )
-        cor = temp;
-    else
-        view._from = view._from.addScaled(new x3dom.fields.SFVec3f(1,0,0), tx);
+        var viewpoint = view._scene.getViewpoint();
+        
+        view._up   = view._flyMat.e1();
+        view._from = view._flyMat.e3(); // eye
+        var s = view._flyMat.e0();                   
+
+        // add xy offset to look-at position  ^
+        cor = view._at;     
                     
-    view._at = cor;
-    
-    // update camera matrix with lookAt() and invert
-    view._flyMat = x3dom.fields.SFMatrix4f.lookAt(view._from, cor, view._up);
-    viewpoint.setViewAbsolute(view._flyMat.inverse());
+        cor = cor.addScaled(this.panAxisX, ty);
+        var temp = cor;
+        if(cor.y > bbox.max.y || cor.y < bbox.min.y )
```

Comment:
I would expect that the x delta on the screen maps to a translation along "panAxisX", not along "panAxisY"